babel-code-frame/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We don't run tests, just typecheck them. So it just needs to be able to import the `babel-code-frame` types (defined in `index.d.ts`).